startercode/routes/index.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+// -- require npm packages
+const express = require('express');
+const router = express.Router();
+
+// -- require your own modules (router, models)
+// const User = require('../models/user');
+
+/* GET home page. */
+
+router.get('/', (req, res, next) => {
+  const username = req.app.locals.username;
```

Comment:
Since you have the following configuration in the app.js you don't need to read it from req.app.locals.username. User.username will display it on your templates
```
app.use((req, res, next) => {
  app.locals.user = req.session.user;
  next();
});
```